downstream/modules/aap-on-azure/proc-azure-update-route-tables.adoc

```diff
@@ -47,10 +47,14 @@ After you have configured the routing rules, traffic is routed to and from {Plat
 
 .Outbound routing through virtual appliances
 
-If your organization uses Azure firewall services or third-party firewall appliances through a Virtual Appliance connection, you must configure routes to enable application management and automation against external resources.
+If your organization uses Azure firewall services or third-party firewall appliances through a Virtual Appliance connection, you must configure outbound connectivity from the managed application, to allow Red Hat to maintain your application and to allow automation against external resources.
```

Comment:
IBM style guide says not to use 'allow', so maybe:
so that Red Hat can manage your application...
and
...a firewall rule that routes all outbound traffic from port 443.

Reply:
As this would be a firewall allow rule, do you think it would be OK in this instance?

Reply:
Because firewalls allow and block traffic, rather than route traffic, as far as I know.

Reply:
Changed "allow" to enable in line 50

Reply:
@ianf77 For the firewall, I think we should stick with "allow" as firewall rules block or allow traffic: they don't route it.